docs/deployment.md

```diff
@@ -70,6 +70,8 @@ Make sure your `package.json` has the `"build"` and `"start"` scripts:
 
 `next build` builds the production application in the `.next` folder. After building, `next start` starts a Node.js server that supports [hybrid pages](/docs/basic-features/pages.md), serving both statically generated and server-side rendered pages.
 
+All Next.js assets are hashed by default. This means if you use a CDN, with every new deployment, a CDN purge wouldn't be required.
```

Comment:
```suggestion
All Next.js assets are hashed by default. This means if you use a CDN that doesn't automatically handle deployment caches like Vercel, with every new deployment, a CDN purge wouldn't be required for files in `/_next` although is still needed for any resources like SSR'd pages outside of the `/_next` folder.
```

We might want to make sure to stress this only applies to `/_next` assets and resources outside of there still might require a purge if not hashed or prevented from being cached